gips/data/landsat/landsat.py

```diff
@@ -286,6 +289,53 @@ def __init__(self, filename):
             raise Exception("Sensor %s not supported: %s" % (self.sensor, filename))
         self._version = self.version
 
+    def filter(self, pclouds=100, **kwargs):
```

Comment:
If this is relatively final, can I ask for a docstring?  Also if `kwargs` isn't used, can it be taken out (unless there's a plan for the future for it maybe)?

Reply:
fixed